src/commcare_cloud/fab/operations/release.py

```diff
@@ -368,6 +371,30 @@ def record_successful_deploy():
         })
 
 
+@roles(ROLES_DEPLOY)
+def publish_dimagi_qa_event(name):
+    if env.deploy_env != "production":
+        return
+    environment = get_environment(env.deploy_env)
+    github_token = environment.get_secret("dimagi_qa_github_token")
+    if not github_token:
+        print(red(f"skipping dimagi-qa {name} event: "
+            "dimagi_qa_github_token secret not set"))
```

Comment:
Is this message is appropriate to show (1) on staging and (2) on third party environments? Especially for third party environments it implies they're doing something wrong, but I think we expect only dimagi to be setting up this automated QA hook, right?

Reply:
This will only be shown if the check above passes (currently `if env.deploy_env != "production"`). I'm going to change that though to make the above check based on a variable set in the environment. Just need to figure out how to do that with fabric.